docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+<!--[metadata]>
++++
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_content_trust"
+weight=-1
++++
+<![end-metadata]-->
+
+# Content trust in Docker
+
+When transferring data among networked systems, *trust* is a central concern.
```

Comment:
I'll tweak.  The docs are technical the feature either is or is not available --- the blog and the announcement are the correct place to say --- we are going to release this for DTR/and private regs in the near future.